### PR TITLE
Aboutpage

### PR DESCRIPTION
**Pull request by oriyannadav**
The about page is now ready and responsive.

I have made a couple more changes to the footer and the home page.

Would love some input and feedback on how it looks and any appropriate changes you think might make it better.

### CODE REVIEW

File: src/pages/AboutPage/AboutPage.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+function AboutPage() {
+
+    return (
+        <section className="section about">
+            <div className="photo">
+                <img src="/mentoring1.jpeg" alt="" />
+            </div>
+            <div className='about-text'>
+                <h1>About US</h1>
```
Comment (rosiejeays) [inline src/pages/AboutPage/AboutPage.jsx]: Is US supposed to be in caps?

File: public/mentoring1.jpeg
Comment (rosiejeays) [inline public/mentoring1.jpeg:1]: It's really tricky to see the nav bar against this picture as the silhouette on the same side as the navbar links

**Review by rosiejeays — APPROVED** [review]
I'm not going to block this, but I would _strongly_ recommend reviewing the picture for the about page.
Also, if "US" in caps is a typo, fix that up before merging to development. :)

File: src/pages/AboutPage/AboutPage.jsx (hunk 1 above)
Comment (oriyannadav) [reply src/pages/AboutPage/AboutPage.jsx]: No the US in capitals is an accident, will change it now.

File: public/mentoring1.jpeg
Comment (oriyannadav) [reply public/mentoring1.jpeg:1]: Not really sure what you mean?

Comment (rosiejeays) [reply public/mentoring1.jpeg:1]: When you hit the more info button it takes you straight to the content of the about page. This means the page is scrolled down and you can't see the nav links:
Screen Shot 2023-10-21 at 11 08 57 am

Comment (oriyannadav) [reply public/mentoring1.jpeg:1]: I have flipped the image horizontally, which looks like this now
Screenshot 2023-10-21 at 11 23 47 am

**Pull request merged**